### PR TITLE
feat: Remove disabling deployment

### LINKED ISSUE #236: Remove disabling deployment
As its the entire design of the chart, it seems fruitless

### PR DESCRIPTION
**Pull request by DirtyCajunRice**
Fixes #236

### CODE REVIEW

**Comment by DirtyCajunRice** [comment]
Seems like a lot but its just a couple spaces fixed with indenting + the deployment removal.

File: traefik/values.yaml [hunk 1]
```diff
@@ -8,7 +8,6 @@ image:
 # Configure the deployment
 #
 deployment:
-  enabled: true
```
Comment (mmatur) [inline traefik/values.yaml:11]: Regarding this PR https://github.com/containous/traefik-helm-chart/pull/110 and this issue https://github.com/containous/traefik-helm-chart/issues/68, could you please keep the `enable` field.

**Review by mmatur — CHANGES_REQUESTED** [review]

File: traefik/values.yaml (hunk 1 above)
Comment (SantoDE) [reply traefik/values.yaml:11]: I agree. `enabled` field should be kept.

Comment (DirtyCajunRice) [reply traefik/values.yaml:11]: The entire premise of #68 is based on a lack of proper helm usage and hook definition. Helm will install CRDs prior to the rest and IngressRoutes only after set properly _even when using helm template_. I strongly disagree with this flag existing at all.

Comment (DirtyCajunRice) [reply traefik/values.yaml:11]: Ill leave it for now because there are so many things that need to be fixed but i would like to circle back to this later

**Pull request closed without merging**